footil/formatting.py

```diff
@@ -288,6 +288,31 @@ def replace_email(email_part: str, old_email: str) -> str:
     return formataddr((name, email_part))
 
 
+def email_to_domain(email: str) -> str:
+    """Extract domain part from email.
+
+    If can't extract domain, empty string is returned instead.
+
+    Args:
+        email: email to extract domain from.
+
+    Returns:
+        email's domain part.
+        str
+
+    """
+    # First parse into name and email parts, so we would not have any
+    # artifacts hanging like `<>` or email name etc.
+    email = parseaddr(email)[1]  # Falsy value returns `('', '')`
+    if email:
+        email_parts = email.split('@', 1)
```

Comment:
Couldn't you reuse `split_force` here?

Reply:
Why would I want to use that? `split_force` is intended to be used when you always need to split into wanted number of parts. Here you dont want that. All you want is to know if second part was split or not. So standard split fits it.

Reply:
Am I wrong, or you could just return `formatting.split_force(email, sep='@ ', maxsplit=1, default='')[1]`?

Reply:
You mean for whole function? For that, no it would not work correctly if multiple `@` are passed. as `split` alternative, it basically does same thing. So both are OK I guess.